recipes/metagem/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "metagem" %}
+{% set version = "1.0.5" %}
+
+package:
+  name: "{{ name }}"
+  version: {{ version }}
+
+source:
+  url: "https://github.com/franciscozorrilla/metaGEM/archive/refs/tags/v1.0.5.tar.gz"
+  sha256: "8ee8375967920fe9fabe276512f1dc2d663a8a85c73f28c534306fd66e4c3068"
+
+build:
+  noarch: generic
+  number: 0
+  script: "{{ PYTHON }} -m pip install --user -vvv memote carveme smetana"
```

Comment:
Why not simply ` script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv`

Reply:
I guess Im not sure how to specify that these 3 packages (memote carveme smetana) need to be installed via pip, while the ones below via conda?

Reply:
Oh I see. No this is not recommended :)
We need to build those three packages at first as conda packages.

Use grayscull to build packages for those 3 dependencies: https://conda-forge.org/blog/posts/2020-03-05-grayskull/

(and if they are not bio-related submit them please to conda-froge https://github.com/conda-forge/staged-recipes/, ping me there, so I can help)

Reply:
ok I see, thanks! I just checked and carveme appears to be on conda already, but will see what I can do about the other two. for now I will leave these tools out of the recipe and add them later